lib/storage.js

```diff
@@ -162,6 +162,10 @@ Storage.prototype._completeTxData = function(walletId, txs, cb) {
       _.each(tx.actions, function(action) {
         action.copayerName = wallet.getCopayer(action.copayerId).name;
       });
+
+      if (tx.status=='accepted')
```

Comment:
Do you think we could check for `broadcasted` as well? This way we could get the raw tx from `getTx()` which could be useful for client apps. Maybe create a `TxProposal.prototype.hasRawTx()` method to check for this condition?